### PR TITLE
DDIDS-953-added-missing-property-descriptions

### PR DESCRIPTION
**Pull request by Radha-Alberta**
Added missing property descriptions as per this [document](https://goa-dio.atlassian.net/wiki/spaces/DS/pages/2450129294/Component+Properties).

### CODE REVIEW

File: libs/docs/src/components/common/Badge.stories.mdx [hunk 1]
```diff
@@ -21,18 +29,14 @@ which display minimal amounts of information, system feedback, or states.
...
-    description="Badge text"
+    description="Define the context and colour of the badge"
   />
+  <Prop name="content" type="string" description="Text label of the badge" />
   <Prop
     name="icon"
     type="boolean"
     defaultValue="false"
-    description="Flag to show the icon corresponding to the defined type"
+    description="Include an icon in the badge"
```
Comment (ArakTaiRoth) [inline libs/docs/src/components/common/Badge.stories.mdx:39]: I'm not sure why you changed it to this? How it was previously is what is in the document you linked to.

File: libs/docs/src/components/common/Badge.stories.mdx [hunk 2]
```diff
@@ -21,18 +29,14 @@ which display minimal amounts of information, system feedback, or states.
...
     required={true}
-    description="Defines the context and colour of the badge"
-  />
-  <Prop
-    name="content"
-    type="string"
-    description="Badge text"
+    description="Define the context and colour of the badge"
   />
+  <Prop name="content" type="string" description="Text label of the badge" />
```
Comment (ArakTaiRoth) [inline libs/docs/src/components/common/Badge.stories.mdx:34]: In the document you linked to, this property should be listed below the icon property.

File: libs/docs/src/components/common/Callout.stories.mdx [hunk 3]
```diff
@@ -25,15 +25,12 @@ import { GoACallout } from '@abgov/react-components';
 <Props>
   <Prop
     name="type"
-    type="emergency | important | information | event | success"
+    type="Information | important | emergency | success | event"
```
Comment (ArakTaiRoth) [inline libs/docs/src/components/common/Callout.stories.mdx:28]: Why did you changed the order of these options? How it was, is how it is in the document you linked. Also, I wouldn't capitalize `information`.

File: libs/docs/src/components/common/Callout.stories.mdx [hunk 4]
```diff
@@ -25,15 +25,12 @@ import { GoACallout } from '@abgov/react-components';
...
     required="true"
-    description="Sets the callout color and side icon"
-  />
-  <Prop
-    name="heading"
-    type="string"
-    description="Callout heading text"
+    defaultValue="Information"
```
Comment (ArakTaiRoth) [inline libs/docs/src/components/common/Callout.stories.mdx:30]: Don't capitalize this, even if it is in the document.

**Comment by ArakTaiRoth** [comment]
You need to update this branch with the latest that's on alpha. You're missing stuff.

File: libs/docs/src/components/common/Callout.stories.mdx (hunk 3 above)
Comment (Radha-Alberta) [reply libs/docs/src/components/common/Callout.stories.mdx:28]: I have changed it according to the Tom's comment. I will change the casing of the information.
image

File: libs/docs/src/components/common/Callout.stories.mdx (hunk 4 above)
Comment (Radha-Alberta) [reply libs/docs/src/components/common/Callout.stories.mdx]: Done.

File: libs/docs/src/components/common/Badge.stories.mdx (hunk 2 above)
Comment (Radha-Alberta) [reply libs/docs/src/components/common/Badge.stories.mdx]: Done.

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**